Add function to convert the old docker check configuration's file to the new format.

hush-hush opened the pull request:
### What does this PR do?

The 'import' command will now read docker_daemon.yaml and generate
docker.yaml. Unsupported information will be dropped with a warning to
the user.

xvello reviewed (commented):
Logic LGTM after the error management change.

Re-reading the code with a clearer head leads me to think the exclude/include settings will also impact AD and tagger, and therefore should be migrated to `datadog.yaml` with `util/docker` directly reading them on init, instead of going through ther docker check.

Does that make sense to you?

xvello reviewed (dismissed):
Let's merge this to move forward, then work on moving these options to datadog.yaml and move that logic to the `datadog.yaml` legacy import logic

hkaj reviewed (dismissed):
LGTM - @masci what do you think about Xavier's comment above? I think it makes sense but I also don't want datadog.yaml to be 2/3 docker stuff...

hush-hush committed:
Revert "Revert "move exclude/include container list to datadog.yaml" (#637)"

This reverts commit afe9d7348d5cac6096fd46d4362cffcd4cb02063.

hush-hush committed:
Migrate docker_daemon.yaml file.

This adds a new option to the 'import' command. Since the new
'ac_include'/'ac_exclude' behaviour could impact users billing, the
option is disabled by default. Users will have to run it manually and
double check the warnings displayed.

The '--docker' option also moves needed information from
docker-daemon.yaml to datadog.yaml.

hush-hush commented:
**New Version**

This adds a new option to the 'import' command. Since the new
'ac_include'/'ac_exclude' behaviour could impact users billing, the
option is disabled by default. Users will have to run it manually and
double check the warnings displayed.

The '--docker' option also moves needed information from
docker-daemon.yaml to datadog.yaml.

hush-hush commented on cmd/agent/app/import.go:
```diff
@@ -128,6 +115,18 @@ func doImport(cmd *cobra.Command, args []string) error {
 		src := filepath.Join(oldConfigDir, "conf.d", f.Name())
 		dst := filepath.Join(newConfigDir, "conf.d", f.Name())
 
+		if f.Name() == "docker_daemon.yaml" {
```
We write it in the same directory, but we should avoid copying the old `docker.yaml` configuration file as it could overwrite the one we just created.
